src/sentry/static/sentry/app/views/settings/organization/members/organizationMembersView.jsx

```diff
@@ -0,0 +1,270 @@
+import {browserHistory} from 'react-router';
+import React from 'react';
+
+import {t, tct} from '../../../../locale';
+import Button from '../../../../components/buttons/button';
+import ConfigStore from '../../../../stores/configStore';
+import IndicatorStore from '../../../../stores/indicatorStore';
+import SentryTypes from '../../../../proptypes';
+import SpreadLayout from '../../../../components/spreadLayout';
+import OrganizationSettingsView from '../../../organizationSettingsView';
+import OrganizationMemberRow from './organizationMemberRow';
+import OrganizationAccessRequests from './organizationAccessRequests';
+
+class OrganizationMembersView extends OrganizationSettingsView {
+  static contextTypes = {
+    organization: SentryTypes.Organization
+  };
+
+  // XXX(billy): setState causes re-render of the entire view...
```

Comment:
what does this comment mean?

Reply:
Updated comment, but to leave it here, since this component is using class inheritance, it'll cause re-renders all the way up to the parent.